### ISSUE TITLE
Johns Hopkins: contradictory record counts

### ISSUE THREAD

**Issue by bobpersing** [pod4lib/aggregator#773]
This relates to the general question in #753, but is more specific:

The default stream for Johns Hopkins has these numbers:
Records: 2,065,546
Unique records: 1,700,.240
There are 27 files in the stream, of which 15 are labelled "Valid MARC".  The individual totals for those 15 add up to 1,700,240.

But the providers index page has these numbers:
Johns Hopkins
Files: 27
Unique records: 0
Total records: 0

I want to go back to Johns Hopkins and ask whether this is a full set of their records.  (They have over 2 million in their local catalog.). Before I do, though, I want to be sure the records they've already sent were successfully ingested.  If they were, why does the providers page list 0 records?

**Comment by corylown** [comment]
@bobpersing these statistics are generated in a background job that runs once a day. I kicked off the job manually for JHU and now the counts are appearing as expected on the providers page. I'll do more digging to see if I can figure out _why_ they weren't generated during the daily job.

**Comment by corylown** [comment]
@bobpersing I found that the organization statistics (the ones shown on the page that lists all the providers) are a day behind the statistics shown on the individual provider/default stream page. This was due to the bug in the stats processing code. I've put in a pull request with a fix for this so the stats should be more consistent once this change is rolled out. We're also considering adding some messaging near the stats to indicate they're calculate daily and do not reflect the most recent uploads.
